### ISSUE TITLE
New Automated Process for Acxiom Data Installs

### ISSUE THREAD

**Issue by charlesdublend360** [BLEND360/innovation_lab#15]
1. Set up scheduler so that these jobs can run automatically. (recommend airflow on AWS or AWS Glue or Databricks scheudler)
2. Use an ETL orchestrator to create script that decrypts files in parallel. 
3. Use Spark job (AWS Glue/EMR/Databricks) to create smaller parquet files for Snowflake ingestion. 
4. Snowflake job to upload new files into tables
5. Archive install files into cold storage.
6. Build automated data quality checks to ensure incoming data doesn't suck.

**Comment by charlesdublend360** [comment]
@moonziyue  will look into airflow/glue first. Loop in Abhiram and Sam

**Comment by charlesdublend360** [comment]
@moonziyue  is this complete yet?

**Comment by moonziyue** [comment]
yes. still wanted to improve the quality of the code when I'm free but we have a working pipeline.
